admin/client/App/screens/Item/components/RelatedItemsList.js

```diff
@@ -96,25 +86,43 @@ const RelatedItemsList = React.createClass({
 			]}</tr>
 		);
 	},
-	render () {
-		let tbody = <tr><td><Spinner size="sm" /></td></tr>;
+	renderItems () {
+		return this.state.items.results.map(this.renderTableRow);
+	},
+	renderSpinner () {
+		return <tr><td><Spinner size="sm" /></td></tr>;
+	},
+	renderNoRelationships () {
+		return (
+			<tr>
+				{this.renderRelationshipColumn({})}
+				{this.renderParentColumn({})}
+				<td>None</td>
+				<td></td>
+				<td></td>
+			</tr>
+		);
+	},
+	renderError () {
+		return <tr><td>{this.state.err}</td></tr>;
+	},
+	renderRelationshipTableBody () {
+		const loadedItems = this.state.items;
+		const results = this.state.items && this.state.items.results;
+		let tbody = null;
 		if (this.state.err) {
-			tbody = <tr><td>{this.state.err}</td></tr>;
-		} else if (this.state.items && this.state.items.results && this.state.items.results.length) {
-			tbody = this.state.items.results.map(this.renderTableRow);
-		} else if (this.state.items && this.state.items.results && !this.state.items.results.length) {
-			tbody = (
-				<tr>
-					{this.renderRelationshipColumn({})}
-					{this.renderParentColumn({})}
-					<td>None</td>
-					<td></td>
-					<td></td>
-				</tr>
-			);
+			tbody = this.renderError();
+		} else if (loadedItems && results.length) {
```

Comment:
results && results.length is the same and more indicative of function. You could also nest this and the next if to show that this section is concerned with results